tests/test_lxc_installer.py

```
@@ -639,3 +650,160 @@ async def test_lxc_uninstall_container_error_blocks_store_removal(self, lxc_clie
+    forced through that Gitea path. Each test pins one piece of the behaviour
+    the generic path MUST provide once it lands. They are xfail(strict=True):
+    green today by failing as expected, and the moment the implementation
+    ships they XPASS, which strict mode flips into a failure that forces the
+    marker to be removed.
+    """
+
+    async def _run_generic_install(
+        self,
+        captured_cmds: list,
+        *,
+        admin_password: str = "secret",
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to function default: "admin_password"

(S107)